src/ocamlorg_frontend/css/styles.css

```diff
@@ -236,10 +236,10 @@ summary::marker {
 /* summary & details expandable section */
 
 details summary svg {
-  transform: rotate(90deg);
+  transform: rotate(0deg);
```

Comment:
There are other instances of `details summary svg`s in the existing templates. All these chevrons need to be modified so that their default orientation is downwards (e.g. replace chevron-right by chevron-down).

Reply:
There was another instance of `chevron_right` used in a `details summary` and it's been corrected in the latest pr